.github/workflows/test_python.yaml

```diff
@@ -31,43 +38,92 @@ jobs:
       - name: Install Doxygen (Windows)
         if: contains(matrix.os, 'windows')
         run: |
-          choco install doxygen.install --version 1.8.20
-          echo "C:\Program Files\doxygen\bin" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
-      ##########################################################################
+          # This command has been failing with source forge mirror issues
+          # choco install doxygen.install --version 1.9.6
```

Comment:
FYI since they were still failing intermittently I just snagged a different installation method.  I think at one point in the past I was doing the doxygen.nl download but switched away.